articles/azure/azs-how-to-obtain-clientid-via-api.md

```diff
@@ -80,23 +80,23 @@ c. ClientID (Application ID)
 
 **Do Not Use - it is legacy and does not have all permissions*
```

Comment:
This has not formatted correctly and is confusing as it's not clear what it's referring to.

Should just be:

* Do not use - this is legacy and does not have all the necessary permissions.

Reply:
Changed to a note and clarfied what it is refering to